win/first-run.bat

```diff
@@ -0,0 +1,26 @@
+rem this script MUST be run before launching hushd for the first time
+mkdir %AppData%\Hush
+mkdir %LocalAppData%\HushSwingWalletUI
+
+@echo off
+
+IF NOT EXIST %AppData%\Hush\hush.conf (
+   (
+	echo addnode=explorer.myhush.org
+	echo addnode=hushipv4.matthewreichardt.com
+	echo addnode=mmc01.madbuda.me
+	echo addnode=stilgar.leto.net
```

Comment:
hushipv4.matthewreichardt.com, node.myhush.network and mmc01.madbuda.me are no longer up, they were run by former core devs.

My server stilgar also has the hostname of `stilgar.myhush.org` now, please use that.

Our DNS Seed host should also be added here: `dnsseed.myhush.org`. Currently it is not working, but I think we can believe it will be working soon, and add it as a node for people. When it does work, it will greatly help new nodes sync to the network.